py/uchecker.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+import os
+import json
+import errno
+import struct
+import logging
+
+from collections import namedtuple
+
+ELF64_HEADER = "<16sHHIQQQIHHHHHH"
+ELF_PH_HEADER = "<IIQQQQQQ"
+ELF_NHDR = "<3I"
+PT_NOTE = 4
+NT_GNU_BUILD_ID = 3
+IGNORED_PATHNAME = ["[heap]", "[stack]", "[vdso]", "[vsyscall]", "[vvar]"]
+
+Range = namedtuple('Range', 'offset size start end')
```

Comment:
AFAIK, kernel calls these `struct vma_struct`. Let's use the name VMA here as well.